data/docmaps/regression_test/docmap_by_manuscript_id/87356.json

```diff
@@ -0,0 +1,707 @@
+                            "published": "2024-01-11T14:00:00+00:00",
+                            "partOf": {
+                                "type": "manuscript",
+                                "doi": "10.7554/eLife.87356",
+                                "identifier": "87356",
+                                "subjectDisciplines": [
+                                    "Neuroscience",
+                                    "Computational and Systems Biology"
+                                ],
+                                "published": "2023-06-26T14:00:00+00:00",
+                                "volumeIdentifier": "12",
+                                "electronicArticleIdentifier": "RP87356",
+                                "complement": []
+                            }
+                        }
+                    ]
+                }
+            ],
+            "assertions": [
+                {
+                    "item": {
+                        "type": "preprint",
+                        "doi": "10.7554/eLife.87356.2",
+                        "versionIdentifier": "2"
+                    },
+                    "status": "manuscript-published"
+                }
+            ],
+            "inputs": [
+                {
+                    "type": "preprint",
+                    "doi": "10.1101/2023.03.24.534142",
+                    "url": "https://www.biorxiv.org/content/10.1101/2023.03.24.534142v2",
+                    "versionIdentifier": "2"
+                },
+                {
+                    "type": "review-article",
+                    "doi": "10.7554/eLife.87356.2.sa0"
+                },
+                {
+                    "type": "review-article",
+                    "doi": "10.7554/eLife.87356.2.sa1"
+                },
+                {
+                    "type": "evaluation-summary",
+                    "doi": "10.7554/eLife.87356.2.sa2"
+                },
+                {
+                    "type": "reply",
+                    "doi": "10.7554/eLife.87356.2.sa3"
+                }
+            ],
+            "next-step": "_:b6",
+            "previous-step": "_:b4"
+        },
+        "_:b6": {
+            "actions": [
+                {
+                    "participants": [],
+                    "outputs": [
+                        {
+                            "type": "version-of-record",
+                            "identifier": "87356",
+                            "doi": "10.7554/eLife.87356.3",
+                            "versionIdentifier": "3",
+                            "published": "2024-02-14",
+                            "url": "https://doi.org/10.7554/eLife.87356.3",
+                            "content": [
+                                {
+                                    "type": "web-page",
+                                    "url": "https://elifesciences.org/articles/87356v1"
```

Comment:
Hi @fred-atherden,
Just wanted to double check we want the url here with version as in the example. Not like the existing url which is `"url": "https://elifesciences.org/articles/87356"` without version?

Reply:
Thanks @HazalCiplak. Yes I think it makes sense to used the versioned URI, as above. That way EPP can link to a specific VOR version (there can be multiple) from the timeline. Let me know if unclear!